compiler/rustc_mir_build/src/builder/scope.rs

```diff
@@ -922,9 +922,9 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
         {
             let this = &mut *self;
             let blocks = drops.build_mir::<ExitScopes>(&mut this.cfg, Some(real_target));
```

Comment:
can we refactor this to call `this.build_exit_tree`? I'd rather not duplicate this kind of complex code.

Reply:
That works too.